superduper/base/leaf.py

```diff
@@ -147,6 +148,7 @@ def metadata(self):
 
     def __post_init__(self, db: t.Optional['Datalayer'] = None):
         self.db = db
+        self.inline = False
```

Comment:
What is this “inline” specifically used for? Could you give an example?

Reply:
`dill_serializer` - these are "hard-coded" components, which don't need to be added to the `db` because they are importable.

Reply:
Great!